const throwError = err => {
   return new Error(err)
}

export default throwError
